test/index.spec.js

```diff
@@ -236,3 +240,63 @@ it('should fulfill the promise the array of expected actions is contained in the
 
     return promise;
 });
+
+it('should teardown correctly when promise fulfills', async () => {
+    const mockStore = createMockStore([thunkMiddleware]);
+    const clearTimeoutSpy = jest.spyOn(global, 'clearTimeout');
+    const getUnsubcribeSpy = spyOnUnsubscribe(mockStore);
+
+    mockStore.dispatch(asyncActionsCreator([action1]));
+
+    await waitForActions(mockStore, action1);
+
+    expect(clearTimeoutSpy).toHaveBeenCalledTimes(1);
+    expect(getUnsubcribeSpy()).toHaveBeenCalledTimes(1);
+});
+
+it('should teardown correctly when promise rejects via timeout', async () => {
+    expect.assertions(1);
+    const mockStore = createMockStore([thunkMiddleware]);
+    const getUnsubcribeSpy = spyOnUnsubscribe(mockStore);
+
+    mockStore.dispatch(asyncActionsCreator([action1]));
+
+    try {
+        await waitForActions(mockStore, [action1, action2]);
+    } catch (err) {
+        expect(getUnsubcribeSpy()).toHaveBeenCalledTimes(1);
+    }
+});
+
+it('should teardown correctly when promise rejects via cancelation', () => {
+    expect.assertions(2);
```

Comment:
Could you add a new line after `expect.assertions`? Do the same everywhere this applies.

Reply:
Sure